site/_resources/release-process.md

```diff
@@ -3,136 +3,269 @@ title: Contour Release Process
+
+### Toot your horn
+
+- Post a note to the #contour channel on k8s slack, also update the /topic with the current release number
+- Post a note to the #project-contour channel on the vmware slack, also update the /topic with the current release number
+- Send an update to the [cncf-contour-users mailing list](https://lists.cncf.io/g/cncf-contour-users/)
+
+### File issues
+
+If you encountered any problems or areas for improvement while executing the release, file issues before you forget.
+
+
+
+# Patch Release Process
 
-## Patch release
+## Overview
 
-### Cherry-pick required commits into the release branch
+A patch release is e.g. `v1.11.1`.
 
-Get any required changes into the release branch by whatever means you choose.
+A patch release requires:
+- patches to be cherry-picked onto the existing release branch
+- website updates
+- YAML to be customized
+- a release tag to be created
+- an Operator release
+- a GitHub release with release notes
+- public communication
+- cleanup
 
-### Tag patch release from release branch
+## Process
 
-Tag the head of your release branch with the release tag, and push
+### Set environment variables
 
-```sh
-$ git tag -a v1.2.1 -m 'contour 1.2.1'
-$ git push --tags
+Set environment variables for use in subsequent steps:
+
+```bash
+export CONTOUR_RELEASE_VERSION=v1.11.1
+export CONTOUR_RELEASE_VERSION_MAJOR=1
+export CONTOUR_RELEASE_VERSION_MINOR=11
+export CONTOUR_PREVIOUS_VERSION=v1.11.0
+
+export CONTOUR_UPSTREAM_REMOTE_NAME=upstream
+export CONTOUR_OPERATOR_UPSTREAM_REMOTE_NAME=upstream
 ```
 
-## Finishing up
+### Cherry-pick relevant commits into release branch
+
+1. Get a list of commit SHAs from `main` to backport.
+1. Check out the release branch for the minor version you're patching (i.e. `release-${CONTOUR_RELEASE_VERSION_MAJOR}.${CONTOUR_RELEASE_VERSION_MINOR}`), ensure it's up to date, and ensure you have a clean working directory.
+1. Create a new local feature branch from the release branch.
+1. Cherry-pick each commit from Step 1, fixing any conflicts as needed:
+
+    ```bash
+    # repeat for each SHA
+    git cherry-pick <SHA>
+    ```
+
+1. Commit all changes, push the branch, and PR it into the release branch.
+
+### Update the website with release-specific information
+
+1. Check out `main`, ensure it's up to date, and ensure you have a clean working directory.
+1. Create a new local feature branch from `main`.
+1. Generate a new set of versioned docs:
+    
+    ```bash
+    go run ./hack/release/prepare-release.go $CONTOUR_PREVIOUS_VERSION $CONTOUR_RELEASE_VERSION
+    ```
+
+1. Add the new release to the compatibility matrix (`/site/_resources/compatibility-matrix.md`).
+1. Document upgrade instructions for the new release (`/site/_resources/upgrading.md`).
+1. Commit all changes, push the branch, and PR it into `main`.
+
+TODO do we want to cherrypick docs into the release branch??
```

Comment:
open question - do we care to to cherrypick the new versioned docs into the release branch? The current instructions don't have us do this. I don't think it really matters since the site is always generated from `main`.

Reply:
Given we would likely never change the site/take a site PR for a previous release branch its probably ok to leave out IMO

Reply:
Could we reverse the steps and do the docs first, then create the release branch?

Reply:
This is for a patch release so we already have a release branch. We could definitely do docs first, in `main`, and then cherrypick them into the release branch, if we care to have them in there.

Reply:
ohh I see, I don't think it's a big deal just for docs.

Reply:
cool, SGTM. we can always decide to do it down the road if we want for some reason.